### ISSUE TITLE
Add a new tutorial for DotNet Core with MongoDB

### ISSUE THREAD

**Issue by prkhandelwal** [aspnet/Docs#8859]
**Help us make content visible**

- Tell us what search terms you used and how you searched docs.
- Tell us what docs you found that didn't address your concern.

**Describe the new topic**

- Explain why this topic is needed : 
 There is no tutorial present for beginners to understand how dotnet core can work with MongoDB.

- Suggest a location in the Table of Contents:
Docs/tree/master/aspnetcore/tutorials

- Write an abstract. In one **short** paragraph, describe what this topic will cover.
Creating a Web API using ASP.NET Core to perform CRUD operations on a MongoDB Collection.

- Create an outline for the new topic. We'll help review the outline and approve it before anyone writes a topic. 
1) Setting up MongoDB
2) Creating Dotnet Core web API Application on VS2017
3) Adding required packages
4) Adding Data Models
5) Writing Class for CRUD Operations
6) Making Controller
7) Building and running the application

**Comment by Rick-Anderson** [comment]
Are you looking for a ASP.NET COre UI + mongo version or a WEB API + mongo version. Googling "DotNet Core with MongoDB" yield several of each.

**Comment by prkhandelwal** [comment]
Yeah, I know googling will yield several results. I jus t think that having it in our official docs is a good idea.
